src/util/getUrl.ts

```diff
@@ -11,7 +11,7 @@ export const getGitHubConfigurationUrl = (urlParams: UrlParams): string => {
 };
 
 export const getJiraMarketplaceUrl = (jiraHost: string): string =>
-	`https://${jiraHost}/plugins/servlet/upm/marketplace/plugins/com.github.integration.production`;
+	`${jiraHost}/plugins/servlet/ac/com.atlassian.jira.emcee/discover/app/com.github.integration.production#!/discover/app/com.github.integration.production`;
```

Comment:
Could you use `/plugins/servlet/ac/com.atlassian.jira.emcee/discover#!/discover/app/com.github.integration.production` instead as it bring them directly to the github for jira app instead of the search screen